app.py

```diff
@@ -7,9 +10,16 @@
 from components.control_bar import layout as control_bar_layout
 from components.image_viewer import layout as image_viewer_layout
 
+USER_NAME = os.getenv("USER_NAME")
+USER_PASSWORD = os.getenv("USER_PASSWORD")
+
+VALID_USER_NAME_PASSWORD_PAIRS = {USER_NAME: USER_PASSWORD}
+
 app = Dash(__name__)
 server = app.server
 
+auth = dash_auth.BasicAuth(app, VALID_USER_NAME_PASSWORD_PAIRS)
```

Comment:
Wondering if we want to make this something like: 

```
auth = dash_auth.BasicAuth(app, VALID_USER_NAME_PASSWORD_PAIRS) if os.getenv("env") != "local" else None
```
So that we can set a `local` mode where we don't have to log in each time while developing.

Reply:
I have added a `DASH_DEPLOYMENT_LOC` variable that steers if authentication is enabled (when that variable is set to `'Local'`) or not.